client/jetpack-cloud/sections/partner-portal/select-partner-key/index.tsx

```diff
@@ -41,20 +43,9 @@ export default function SelectPartnerKey() {
 		}
 	}, [ hasKey ] );
 
-	const keys = map( partners, ( partner ) =>
-		map( partner.keys, ( key ) => (
-			<Card key={ key.id } className="select-partner-key__card">
-				<div className="select-partner-key__key-name">{ key.name }</div>
-				<Button primary onClick={ () => dispatch( setActivePartnerKey( key.id ) ) }>
-					{ translate( 'Use' ) }
-				</Button>
-			</Card>
-		) )
-	);
-
 	return (
 		<>
```

Comment:
This is interesting. Why do we have an empty react element here instead of using something like a `<div>`?

Reply:
A div is unnecessary in this case. The Query component renders nothing and I wanted to hoist it at the top for visibility hence the empty (fragment) component. Alternatively, I can nest it at the top of `<Main>` and drop the `<></>`.